lib/dry/system/plugins/dependency_graph.rb

```diff
@@ -12,34 +11,43 @@ module DependencyGraph
         def self.extended(system)
           super
 
-          system.use(:notifications)
+          system.instance_eval do
+            use(:notifications)
 
-          system.setting :ignored_dependencies, default: []
+            setting :dependency_graph do
+              setting :ignored_dependencies, default: []
+            end
 
-          system.after(:configure) do
-            self[:notifications].register_event(:resolved_dependency)
-            self[:notifications].register_event(:registered_dependency)
-
-            strategies(Strategies)
+            after(:configure) do
+              self[:notifications].register_event(:resolved_dependency)
+              self[:notifications].register_event(:registered_dependency)
+            end
           end
         end
 
         # @api private
         def self.dependencies
-          {'dry-events': "dry/events/publisher"}
+          {"dry-events" => "dry/events/publisher"}
+        end
+
+        # @api private
+        def injector(**options)
+          super(**options.merge(strategies: DependencyGraph::Strategies))
```

Comment:
`super(**options, strategies: DependencyGraph::Strategies)`

Reply:
Oh yes, will adjust directly in master :)